bugsnag-plugin-android-ndk/src/main/jni/safejni.c

```diff
@@ -44,7 +46,9 @@ jmethodID bsg_safe_get_method_id(JNIEnv *env, jclass clz, const char *name,
     return NULL;
   }
   jmethodID methodId = (*env)->GetMethodID(env, clz, name, sig);
-  if (bsg_check_and_clear_exc(env) == exception_thrown) {
+  if (bsg_check_and_clear_exc(env) == exception_thrown || methodId == NULL) {
+    BUGSNAG_LOG("Could not find method \"%s\" with signature \"%s\"", name,
```

Comment:
I put them in as a sanity check for us in case we misspell a method name. Right now when that happens we no-op, but there's no notification that something went wrong. If extreme memory pressure causes log printing to fail, and it happens in this place, we'd see a crash here. But that's also true of every other place in the codebase where we allocate memory or make a call to something that might allocate.